types/window.ts

```diff
@@ -27,3 +28,11 @@ export interface ShowOpenDialogParams {
 export interface ShowOpenDialogResult {
     uris: URI[]
 }
+
+export interface CheckDiagnosticsParams {
+    filePaths: Record<string, any>
```

Comment:
Is this too generic? Can you give some examples please

Reply:
Is it possible to avoid using `any` here?

And also, can we use a different name? When I see `filePaths` I expect the type to be `string[]`